instruqt-tracks/nomad-integration-with-vault/track.yml

```diff
@@ -0,0 +1,490 @@
+    ```
+    nomad run /root/hashistack/nomad/db.nomad
+    ```
+    Verify the job is running with the following command;<br /><br />
+    ```
+    nomad status database
+    ```
+    You should see a summary of the status command. You can also, click on the <b>"Nomad"</b> tab to explore the job details.<br /><br />
+    Click the <b>"Check"</b> button in the lower right to complete the challenge.
+  notes:
+  - type: text
+    contents: |-
+      <h1>Deploy a Database</h1>
+      <hr />
+      <p>The next few steps will involve configuring a connection between Vault and our database, so let's deploy one that we can connect to.</p>
+  tabs:
+  - title: Server
+    type: terminal
+    hostname: hashistack-server
+  - title: Files
+    type: code
+    hostname: hashistack-server
+    path: /root/hashistack/
+  - title: Nomad
+    type: service
+    hostname: hashistack-server
+    port: 4646
+  - title: Consul
+    type: service
+    hostname: hashistack-server
+    port: 8500
+  - title: Vault
+    type: service
+    hostname: hashistack-server
+    port: 8200
+  difficulty: basic
+  timelimit: 300
+- slug: configure-the-vault-database-secrets-engine
+  id: sd1xbvy07avo
+  type: challenge
+  title: Configure the Vault Database Secrets Engine
+  teaser: Allow Vault to Connect to Our Database and Create Users with Specific Privileges
+  assignment: |-
+    Lets start off by enabling the Database Secrets Engine by running the following command in the <b>"Server"</b> tab;
+    ```
+    vault secrets enable database
+    ```
+    You should see message indicating success.<br /><br />
+    A database connection file has been created for you and you can take a look at it by clicking on the <b>"Files"</b> tab and navigate to <b>hashistack > vault > connection.json</b> to take a look at the contents.<br /><br />
+    Run the following command to configure the connection between the database secrets engine and our database;<br /><br />
+    ```
+    vault write database/config/postgresql @connection.json
+    ```
+    If the operation is successful, there will be no output.<br /><br />
+    Click on the <b>"Files"</b> tab and navigate to <b>hashistack > vault > accessdb.sql</b>. Recall from the previous step that we specified accessdb in the allowed_roles key of our connection information.<br /><br />
+    Run the following command to create the role;
+    ```
+    vault write database/roles/accessdb db_name=postgresql creation_statements=@accessdb.sql default_ttl=1h max_ttl=24h
+    ```
+    You should see message indicating success.<br /><br />
+    Finally, lets generate the PostgreSQL credentials using the following command;
+    ```
+    vault read database/creds/accessdb
+    ```
+    You should see a table listing a user name a password.<br /><br />
+    A policy file file has been created for you and you can take a look at it by clicking on the <b>"Files"</b> tab and navigating to <b>hashistack > vault > access-tables-policy.hcl</b>.</ br></ br>
```

Comment:
I would explain how this policy will ultimately be used and why we are creating it.  (to restrict who can request dynamic credentials from the database secrets engine)

Reply:
I think you did explain in one of the notes screens.